src/main/java/Interactors/GameCreation.java

```diff
@@ -0,0 +1,167 @@
+                        }
+
+                    }
+                case 1:
+                    // playerPositions
+                    for (String[] position : gameData.get(i)) {
+                        playerPositions.put(getOwner(players, position[0]),Integer.parseInt(position[1]));
+                    }
+                    //TODO create saved Tree with savedTree and treeOptions from TextFileTranslator
+                case 2:
+                    // savedTree
+
+                case 3:
+                    // treeOptions
+            }
+        }
+        List<Cell> cells = createCells(properties, standardProperties);
+
+        return new Board(players, cells, playerPositions);
+    }
+    private Player getOwner(ArrayList<Player> players, String playerName) {
+        for (Player player : players) {
+            if (player.getName().equals(playerName)) {
+                return player;
+            }
+        }
+        return null;
+    }
+    private Property getProperty(ArrayList<Cell> standardProperties, String propertyName) {
+        for (Cell property: standardProperties) {
+            Property cProperty = (Property) property;
+            if (cProperty.getName().equals(propertyName)){
+                return cProperty;
+            }
+        }
+        return null;
+    }
+    private List<Cell> createCells(ArrayList<Cell> propertiesSoFar, ArrayList<Cell> standardProperties) throws IOException {
+        CornerTiles go = new CornerTiles("Go");
+        CornerTiles visitingJail = new CornerTiles("justVisiting");
+        CornerTiles inJail = new CornerTiles("inJail");
+        CornerTiles freeParking = new CornerTiles("freeParking");
+        CornerTiles goJail = new CornerTiles("goToJail");
+        ActionSpace communityChest = new ActionSpace("communityChest");
+        ActionSpace chance = new ActionSpace("chance");
+        // TODO: determine when to call jail ActionSpaces
+        // ActionSpace jail = new ActionSpace("jail");
+
+        ArrayList<Cell> cells = standardProperties;
+        cells.add(0, go);
+        cells.add(2, communityChest);
+        cells.add(4, chance);
+        cells.add(7, chance);
+        cells.add(10, visitingJail);
+        cells.add(11, inJail);
+        cells.add(18, communityChest);
+        cells.add(21, freeParking);
+        cells.add(23, chance);
+        cells.add(31, goJail);
+        cells.add(34, communityChest);
+        cells.add(37, chance);
+        cells.add(39, communityChest);
+
+        for (Cell property : propertiesSoFar){
+            Property cProperty = (Property) property;
+            for (int i=0; i <= 41; i++) {
+                if (cells.get(i) instanceof Property){
+                    Property currentCell = (Property) cells.get(i);
+                    if (currentCell.getName().equals(cProperty.getName()));
+                    cells.remove(i);
+                    cells.add(i, cProperty);
+                }
+            }
```

Comment:
Should we separate this chunk into a different method? We might not need this for the case where we create a new game.